changelog/1.get_merged_prs.py

```diff
@@ -35,19 +35,20 @@ def parse_isoformat(string):
 # crashes, we still write out what we had.
 
 try:
-
+    max_issues = repo.get_issues(state='all',sort='created')[0].number
+    i = 0
     for pr in repo.get_pulls(state='closed', sort='updated', direction='desc', base='master'):
-
+        i += 1
```

Comment:
I thought about that, but I think in py 2.7 (and maybe 3?) that lists out all of the for loop so you don't get the live update (because each step in the iteration requires accessing github).  That might not actually be true, but it seemed safer to do this and I was in a hurry ;)